docs/src/developers_guide/release.rst

```diff
@@ -11,6 +11,20 @@ The summary below is of the main areas that constitute the release.  The final
 section details the :ref:`iris_development_releases_steps` to take.
 
 
+.. _release-manager:
```

Comment:
This is just a preference, but I suggest using underscores and not hypens, I think most other doc bookmarks use underscores too.